engine/src/main/java/org/terasology/logic/debug/MovementDebugCommands.java

```diff
@@ -233,22 +236,48 @@ public String stepHeight(@Sender EntityRef client, @CommandParam("height") float
         return "";
     }
 
+    private float getJumpSpeed(float amount){
+        double d = (double)amount;
+        return (float) pow(d,0.6) * 9.36f ;
+    }
+    
+    private float getInteractionRange(float amount){
+        double d = (double)amount;
+        return (float) pow(d,0.6) * 3.9f ;
```

Comment:
The default interaction range seems to be 5 (see [CharacterComponent.java#L37](https://github.com/MovingBlocks/Terasology/blob/develop/engine/src/main/java/org/terasology/logic/characters/CharacterComponent.java#L37)). With this formula the interaction range for height of 1.6 is 5.17055 (≠ 5).

This is what the plot looks like:
![image](https://user-images.githubusercontent.com/1448874/78276579-84bc9000-7513-11ea-8600-1694149e43b6.png)
see [Wolfram Alpha](https://www.wolframalpha.com/input/?i=pow%28x%2C0.6%29+*+3.9%2C+x+in+%5B0%2C5%5D)